src/Frame.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+function Frame() {
+  this.strike = false;
+  this.spare = false;
+  this.frameScore = {roll1: 0, roll2: 0};
```

Comment:
Suggest considering changing this to an array, it's more optimized for performance and more adaptable (e.g. when you need to account for three rolls in the final frame)